.github/workflows/key4hep.yml

```diff
@@ -34,7 +34,7 @@ jobs:
             -DCMAKE_CXX_STANDARD=20 \
             -DCMAKE_CXX_FLAGS=" -fdiagnostics-color=always -Werror -Wno-error=deprecated-declarations " \
             -DUSE_EXTERNAL_CATCH2=AUTO \
-            -DENABLE_RNTUPLE=ON \
+            -DENABLE_RNTUPLE=OFF \
```

Comment:
As soon as we have a recent enough ROOT version in Key4hep this should be switched on again.